consts/consts.go

```diff
@@ -0,0 +1,13 @@
+package consts
+
+const (
+	UserName   = "username"
+	Id         = "id"
+	Author     = "author"
+	Title      = "title"
+	Index      = "books"
+	Price      = "price"
+	PriceRange = "price_range"
```

Comment:
Do those fields indicate on fieldName\ paramsNames? if so - please create more readable packages for those consts